README.md

```diff
@@ -102,3 +102,26 @@ Local JMX is currently enabled on the project. In order to utilize it please fol
 6. Find the health endpoint and click on it to view its attributes and operations. You can use the attributes and operations to check the health of the login-gateway and perform management tasks.
 You can now use JConsole to monitor and manage your local application by accessing the available endpoints.
 Remote JMX is also an option and may be enabled with some config changes in the application.properties file.
+
+## Info endpoint
+Springboot Actuator is enabled for this project. This enables an Info endpoint that can be populated with various information that may be
+useful to the troubleshooting and usage of the application. The endpoint can be accessed via the following url:  `http://localhost:port/actuator/info`.
+The information types available are as follows:
+
+| ID     | Usage                                                        |
+|--------|--------------------------------------------------------------|
+ | build  | Exposes build information                                    |
+| env    | Exposes any property marked under "info" in the config       |
+ | git    | Exposes git related information from the git.properties file |
+| java   | Exposes Java runtime information                             |
```

Comment:
I think the table does not make sense on its own here. For example, the user cannot deduce that for java-related info to be included, they should add `management.info.java.enabled=true` (which works).

Either remove the table altogether or extend it by explaining what to do in the config to gain the extra info.

I'd probably remove it and maybe extend the example config yaml with the `java` info as a comment the user can add.

Reply:
Done so. Removed table and added comments to example config on what each property does